app/code/Magento/Store/Model/Config/Importer/Processor/Create.php

```diff
@@ -177,8 +177,8 @@ private function createGroups(array $items, array $data)
             );
 
             $group = $this->groupFactory->create();
-            $group->setData($groupData);
             $group->setRootCategoryId(0);
+            $group->setData($groupData);
```

Comment:
@larsroettig `setData` with array argument will replace all data including `root_categroy_id` so previous line makes no sense. Maybe it makes sense to define the default value for `root_categry_id` in `$groupData` if it is not set?

Reply:
@slavvka in the most projects is the **root_category_id** not in the groupData this was reasons for switching the lines.  And if you set $groupData **root_category_id** that should overwrite the default value.

Reply:
Look, we have setting default value for root_category_id anyway. But after exchanging the lines the call of stCategoryRootId makes no sense anymore since it will be overwritten by setData. That’s why setting it to default value in case it wasn’t set in groupData preserves the original behavior. Does it make sense?

Reply:
@slavvka no problem I will change this PR to

```php
if(!isset($groupData['root_categry_id'])
{
   $groupData['root_categry_id'] = 0;
}
```